### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by nikitin-da**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+## Version 2.0.0
+
+_2017_09_12_
+
+* **Interceptors API!** :tada::tada: Many thanks to [@rsinukov](https://github.com/rsinukov)
+* Logging via interceptors (just add `LoggingInterceptor`).
+* Remove deprecated `createObservable` and `internal`. You should use `asRxObservable` and `lowLevel` instead.
+* `mapFromCursor` receives `StorIOSqlite`/`StorIOContentResolver` as parameter.
+* Remove `Query.CompleteBuilder.whereArgs(list)`. Please use vararg overload instead.
+* Add ability to use vals instead of vars in classes for resolver generation.
```
Comment (artem-zinnatullin) [inline CHANGELOG.md]: nit: Kotlin

**Comment by artem-zinnatullin** [comment]
>codecov/project — No report found to compare against

Not critical, merging. Sorry it took me so long :(

**Comment by artem-zinnatullin** [comment]
Hm, gpg signing fails with NPE, expect delays in publishing…

**Comment by artem-zinnatullin** [comment]
Uhh, forgot to declare one of the env variables facepalm.jpg

**Comment by artem-zinnatullin** [comment]
Should be available in 10-30 minutes!

**Comment by geralt-encore** [comment]
Is this version live? At least I don't see a new release in GitHub...

**Pull request merged**